src/shared/components/mutationMapper/LegendColorCodes.tsx

```diff
@@ -0,0 +1,192 @@
+import { action, makeObservable, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { DEFAULT_PROTEIN_IMPACT_TYPE_COLORS } from 'react-mutation-mapper';
+import * as React from 'react';
+import { AnnotatedMutation } from 'shared/model/AnnotatedMutation';
+
+interface ILegendColorCodesProps {
+    isPutativeDriver?: (mutation: Partial<AnnotatedMutation>) => boolean;
+    hideFusions?: boolean;
+}
+
+export const LegendColorCodes: React.FC<ILegendColorCodesProps> = observer(
+    ({ isPutativeDriver, hideFusions }: ILegendColorCodesProps) => {
+        return (
+            <div style={{ maxWidth: 700, marginTop: 5 }}>
+                <strong style={{ color: '#2153AA' }}>Color Codes</strong>
```

Comment:
seems like a random color, where'd it come from?

Reply:
Looks like just copied from https://github.com/cBioPortal/cbioportal-frontend/pull/4524/files#diff-9bd4303c188b708b14d30d594c57fa8e7bf773af0903ad255fdde92b29a40750L118. It might be originally coming from the legacy mutation mapper library. https://github.com/cBioPortal/mutation-mapper/blob/5534eadc39ef6396833a16c57a959fbbef88ab00/src/css/mutation_details.css#L121. I don't remember how we pick those colors back then. Might be a random pick.